packages/test-commons/src/generateIsomorphicTests.ts

```diff
@@ -54,6 +58,21 @@ export function generateIsomorphicTests(
                         // errors will fail the test and log full stack traces to the console. nifty!
                         Enzyme.render(element);
                     });
+                    it(`<${componentName}> className is applied to outer element`, () => {
+                        const element = React.createElement(
+                            Component,
+                            {
+                                ...props[componentName],
+                                className: "test",
+                            },
+                            children[componentName],
+                        );
+                        if (classNameChildList.includes(componentName)) {
+                            assert.isAtLeast(Enzyme.shallow(element).find(".test").length, 1);
```

Comment:
❌  `shallow` and `render` are very different... we want to use `render` only here as it produces a DOM tree, whereas `shallow` produces some Enzyme-only tree that doesn't actually assert server-side-rendering success. should be possible to simply replace the word `shallow` with `render`, it still has a `find` method.